uw_spotseeker/__init__.py

```diff
@@ -35,52 +34,40 @@ class Spotseeker(object):
     def post_image(self, spot_id, image: bytes) -> bytes:
         url = "/api/v1/spot/%s/image" % spot_id
 
-        headers = {"X-OAuth-User": settings.OAUTH_USER}
+        headers = {"X-OAuth-User": os.getenv('OAUTH_USER')}
         headers['files'] = {'image': ('image.jpg', image)}
 
-        try:
-            response = Spotseeker_DAO().postURL(url, headers)
-            status = response.status
-            if status not in [200, 201]:
-                raise DataFailureException(url,
-                                           status,
-                                           getattr(response, 'reason', None))
-        except AttributeError:
-            raise NotConfigured("must set credential in settings")
```

Comment:
This isn't necessarily a cause for an AttributeError being raised, so I thought it would be best to pass up the error. However, our settings.py's all set a default for CREDENTIAL so this wouldn't trigger an attribute error.